### PR TITLE
Update cURL examples

### PR DESCRIPTION
**Pull request by danieltprice**
Update cURL examples in the docs
- Do not use -X where it is not necessary
- Use -H for response type

Fix a title linking issue in one of the same files.

https://websitemain62807-dpriceupdateapiexamples.gatsbyjs.io/docs/reference/api-reference/
https://websitemain62807-dpriceupdateapiexamples.gatsbyjs.io/docs/manage/api-keys/
https://websitemain62807-dpriceupdateapiexamples.gatsbyjs.io/docs/manage/branches/#branching-with-the-neon-api

### CODE REVIEW

File: content/docs/manage/api-keys.md [hunk 1]
```diff
@@ -41,16 +41,17 @@ To revoke an API key:
 
 The following `cURL` example uses the `/projects` endpoint to retrieve projects that belong to your Neon account.
 
-```bash
-curl -X GET -H "Authorization: Bearer $NEON_API_KEY" "accept: application/json"
-"https://neon.tech/api-reference/v2" | jq
+```curl
```
Comment (bayandin) [inline content/docs/manage/api-keys.md:44]:
```suggestion
```bash
```

File: content/docs/manage/api-keys.md [hunk 2]
```diff
@@ -41,16 +41,17 @@ To revoke an API key:
...
+curl 'https://console.neon.tech/api/v2/projects' \
+  -H 'accept: application/json' \
```
Comment (bayandin) [inline content/docs/manage/api-keys.md:46]:
```suggestion
  -H 'Accept: application/json' \
```

Headers are case-insensitive, but I think it looks neat when they are capitalised consistently.

File: content/docs/manage/api-keys.md [hunk 3]
```diff
@@ -41,16 +41,17 @@ To revoke an API key:
...
+  -H 'Authorization: Bearer $NEON_API_KEY' | jq
 ```
 
 where:
 
+- `"https://neon.tech/api-reference/v2/projects"` is the resource URL, which includes the base URL for the Neon API and the `/projects` endpoint.
+- The `"accept: application/json"` in the header specifies the accepted response type.
```
Comment (bayandin) [inline content/docs/manage/api-keys.md:53]:
```suggestion
- The `"Accept: application/json"` in the header specifies the accepted response type.
```

File: content/docs/manage/branches.md [hunk 4]
```diff
@@ -108,21 +108,20 @@ Branch actions performed in the Neon Console can be performed using the [Neon AP
...
 The API method appears as follows when specified in a cURL command:
 
-```bash
-curl -X 'POST' \
-  'https://console.neon.tech/api/v2/projects/<project_id>/branches' \
+```curl
+curl 'https://console.neon.tech/api/v2/projects/autumn-disk-484331/branches' \
   -H 'accept: application/json' \
-  -H 'Authorization: Bearer $NEON_API_KEY' \
+  -H 'Authorization: Bearer $NEON_API' \
```
Comment (bayandin) [inline content/docs/manage/branches.md:124]:
```suggestion
  -H 'Authorization: Bearer $NEON_API_KEY' \
```

Should we keep it consistent with `content/docs/manage/api-keys.md`?

File: content/docs/manage/branches.md [hunk 5]
```diff
@@ -108,21 +108,20 @@ Branch actions performed in the Neon Console can be performed using the [Neon AP
...
 ```
 
 The API method appears as follows when specified in a cURL command:
 
-```bash
-curl -X 'POST' \
-  'https://console.neon.tech/api/v2/projects/<project_id>/branches' \
+```curl
+curl 'https://console.neon.tech/api/v2/projects/autumn-disk-484331/branches' \
   -H 'accept: application/json' \
```
Comment (bayandin) [inline content/docs/manage/branches.md:123]:
```suggestion
  -H 'Accept: application/json' \
```

File: content/docs/manage/branches.md [hunk 6]
```diff
@@ -131,38 +130,40 @@ curl -X 'POST' \
     }
   ],
   "branch": {
-    "parent_id": "<parent_id>"
+    "parent_id": "br-wispy-dew-591433"
   }
-}'
+}' |jq
```
Comment (bayandin) [inline content/docs/manage/branches.md:135]:
```suggestion
}' | jq
```

File: content/docs/manage/branches.md [hunk 7]
```diff
@@ -173,52 +174,51 @@ The response includes information about the branch, the branch's endpoint, and t
...
 The API method appears as follows when specified in a cURL command:
 
-```bash
-curl -X 'GET' \
-  'https://console.neon.tech/api/v2/projects/sparkling-king-781971/branches' \
+```curl
+curl 'https://console.neon.tech/api/v2/projects/sparkling-king-781971/branches' \
   -H 'accept: application/json' \
-  -H 'Authorization: Bearer $NEON_API_KEY'
+  -H 'Authorization: Bearer $NEON_API_KEY' |jq
```
Comment (bayandin) [inline content/docs/manage/branches.md:221]:
```suggestion
  -H 'Authorization: Bearer $NEON_API_KEY' | jq
```

File: content/docs/manage/branches.md [hunk 8]
```diff
@@ -227,45 +227,47 @@ The response lists the project's root branch and any child branches.
...
 
-```bash
+```text
 DELETE /branches/{branch_id}
 ```
 
 The API method appears as follows when specified in a cURL command:
 
-```bash
+```curl
```
Comment (bayandin) [inline content/docs/manage/branches.md:270]:
```suggestion
```bash
```

File: content/docs/reference/api-reference.md [hunk 9]
```diff
@@ -23,21 +23,22 @@ The Neon API uses API keys to authenticate requests. You can view and manage API
 The client must send an API key in the Authorization header when making requests, using the bearer authentication scheme. For example:
 
 ```curl
-curl -X GET -H "Authorization: Bearer $NEON_API_KEY" "accept: application/json"
-"https://console.neon.tech/api/v2/projects"
+curl 'https://console.neon.tech/api/v2/projects' \
+  -H 'accept: application/json' \
```
Comment (bayandin) [inline content/docs/reference/api-reference.md:27]:
```suggestion
  -H 'Accept: application/json' \
```

File: content/docs/reference/api-reference.md [hunk 10]
```diff
@@ -23,21 +23,22 @@ The Neon API uses API keys to authenticate requests. You can view and manage API
...
+  -H 'Authorization: Bearer $NEON_API_KEY' | jq
 ```
 
 ## Neon API base URL
 
 The base URL for a Neon API request is:
 
-```text
+```http
```
Comment (bayandin) [inline content/docs/reference/api-reference.md:35]:
```suggestion
```text
```

Is there custom formatting for `http` or `curl` blocks? If not, then they should be `text` and `bash` probably.

File: content/docs/reference/api-reference.md [hunk 11]
```diff
@@ -23,21 +23,22 @@ The Neon API uses API keys to authenticate requests. You can view and manage API
...
 https://console.neon.tech/api/v2/
 ```
 
 Append a Neon API method path to the base URL to construct the full URL for a request. For example:
 
-```text
+```http
```
Comment (bayandin) [inline content/docs/reference/api-reference.md:41]:
```suggestion
```text
```

**Comment by danieltprice** [comment]
Thank you for the review @bayandin. All suggestions accepted.

File: content/docs/manage/branches.md (hunk 6 above)
Comment (bayandin) [reply content/docs/manage/branches.md:135]: @danieltprice one more comment here 😄

File: content/docs/reference/api-reference.md [hunk 12]
```diff
@@ -23,8 +23,9 @@ The Neon API uses API keys to authenticate requests. You can view and manage API
 The client must send an API key in the Authorization header when making requests, using the bearer authentication scheme. For example:
 
 ```curl
-curl -X GET -H "Authorization: Bearer $NEON_API_KEY" "accept: application/json"
-"https://console.neon.tech/api/v2/projects"
+curl 'https://console.neon.tech/api/v2/projects' \
+  -H 'Accept: application/json' \
+  -H 'Authorization: Bearer $NEON_API_KEY' | jq
```
Comment (bayandin) [inline content/docs/reference/api-reference.md:28]:
```suggestion
  -H 'Accept: application/json' \
  -H 'Authorization: Bearer $NEON_API_KEY' \
  -H 'Content-Type: application/json' \
```

Missed `Content-Type` header

**Review by bayandin — APPROVED** [review]
Looks good @danieltprice, I left a couple of more comments.
Thanks!

**Review by bayandin — APPROVED** [review]

**Pull request merged**